gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/ITBigQueryTest.java

```diff
@@ -0,0 +1,782 @@
+      .build();
+  private static final String LOAD_FILE = "load.csv";
+  private static final String JSON_LOAD_FILE = "load.json";
+  private static final String EXTRACT_FILE = "extract.csv";
+  private static final String BUCKET = RemoteGcsHelper.generateBucketName();
+  private static final TableId TABLE_ID = TableId.of(DATASET, "testing_table");
+  private static final String CSV_CONTENT = "StringValue1\nStringValue2\n";
+  private static final String JSON_CONTENT = "{"
+      + "\"TimestampField\": \"2014-08-19 07:41:35.220 -05:00\","
+      + "\"StringField\": \"stringValue\","
+      + "\"IntegerField\": [\"0\", \"1\"],"
+      + "\"BooleanField\": \"false\","
+      + "\"RecordField\": {"
+      + "\"TimestampField\": \"1969-07-20 20:18:04 UTC\","
+      + "\"StringField\": null,"
+      + "\"IntegerField\": [\"1\",\"0\"],"
+      + "\"BooleanField\": \"true\""
+      + "}"
+      + "}\n"
+      + "{"
+      + "\"TimestampField\": \"2014-08-19 07:41:35.220 -05:00\","
+      + "\"StringField\": \"stringValue\","
+      + "\"IntegerField\": [\"0\", \"1\"],"
+      + "\"BooleanField\": \"false\","
+      + "\"RecordField\": {"
+      + "\"TimestampField\": \"1969-07-20 20:18:04 UTC\","
+      + "\"StringField\": null,"
+      + "\"IntegerField\": [\"1\",\"0\"],"
+      + "\"BooleanField\": \"true\""
+      + "}"
+      + "}";
+
+  private static BigQuery bigquery;
+  private static Storage storage;
+
+  @BeforeClass
+  public static void beforeClass() throws IOException, InterruptedException {
+    RemoteBigQueryHelper bigqueryHelper = RemoteBigQueryHelper.create();
+    RemoteGcsHelper gcsHelper = RemoteGcsHelper.create();
+    bigquery = bigqueryHelper.options().service();
+    storage = gcsHelper.options().service();
+    storage.create(BucketInfo.of(BUCKET));
+    storage.create(BlobInfo.builder(BUCKET, LOAD_FILE).contentType("text/plain").build(),
+        CSV_CONTENT.getBytes(StandardCharsets.UTF_8));
+    storage.create(BlobInfo.builder(BUCKET, JSON_LOAD_FILE).contentType("application/json").build(),
+        JSON_CONTENT.getBytes(StandardCharsets.UTF_8));
+    DatasetInfo info = DatasetInfo.builder(DATASET).description(DESCRIPTION).build();
+    bigquery.create(info);
+    JobInfo job = LoadJobInfo.builder(TABLE_ID, "gs://" + BUCKET + "/" + JSON_LOAD_FILE)
+        .createDisposition(JobInfo.CreateDisposition.CREATE_IF_NEEDED)
+        .schema(TABLE_SCHEMA)
+        .formatOptions(FormatOptions.json())
+        .build();
+    job = bigquery.create(job);
+    while (job.status().state() != JobStatus.State.DONE) {
+      Thread.sleep(1000);
+      job = bigquery.getJob(job.jobId());
+    }
+    assertNull(job.status().error());
+  }
+
+  @AfterClass
+  public static void afterClass() throws ExecutionException, InterruptedException {
+    if (bigquery != null) {
+      RemoteBigQueryHelper.forceDelete(bigquery, DATASET);
+    }
+    if (storage != null && !RemoteGcsHelper.forceDelete(storage, BUCKET, 10, TimeUnit.SECONDS)) {
```

Comment:
I just realized that `RemoteGcsHelper.forceDelete` returns Boolean instead of `boolean`.
We should probably change it.